packages/core/src/components/card/card.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+import { safeInvoke } from "../../common/utils";
+
+export type ElevationSize = 0 | 1 | 2 | 3 | 4;
```

Comment:
Should I introduce object witch contains these values? Something like `Intent` or `Classes`

Reply:
I have no problem with the simple number union - it's the same thing I do for `iconSize` in `Icon`. Truth is that your `enum Elevation` above is equivalent and might still support the raw number usage.

Reply:
Let's keep the numbers way in case we update elevations levels. More descriptive key names is too restrictive